src/main/java/com/github/elenterius/biomancy/util/FleshTongue.java

```diff
@@ -26,6 +27,7 @@ private FleshTongue() {}
 		createItemComment(ModItems.AGEING_SERUM.get(), "We can regard our life as a uselessly disturbing episode in the blissful repose of nothingness.\n- Arthur Schopenhauer");
 		createItemComment(ModItems.BREEDING_STIMULANT.get(), "Happiness and the absurd are two sons of the same earth. They are inseparable.\n- Albert Camus");
 		createItemComment(ModItems.SHRINKING_SERUM.get(), "Normalize the idea of living inside of Someone.\n- Tarael Blackwing");
+		createItemComment(Items.CAULDRON,"I've been sprinting towards the horizon of insanity since birth.\n- kd8lvt");
```

Comment:
As we discussed on Discord I don't want to append obfuscated tool-tip text to vanilla Items.
I think the acolyte armor leggings or cradle block are a great option for your quote.